<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkeletonRigWindow</class>
 <widget class="QMainWindow" name="SkeletonRigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>209</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>骨骼设置面板</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SekeletonRigButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>生成骨骼设置</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>191</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>设置骨骼标签</string>
    </property>
    <widget class="QPushButton" name="shouldButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>shouder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="handButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>hand</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cleanSelLabelButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>清除选择的骨骼标签</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cleanAllLabelButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>清除所有的骨骼标签</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>23</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>骨骼默认只生成FK控制器，
为骨骼设置标签可以使骨骼
同时生成IK和FK控制器，
shoulder为IK起始端，
hand为IK末端
(注意：根骨骼不能设置这两
个标签)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>299</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>选择根骨骼再执行以下操作</string>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>SekeletonRigButton</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>209</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
